medusa/providers/torrent/json/t411.py

```diff
@@ -92,20 +188,38 @@ def search(self, search_strings, age=0, ep_obj=None):
 
             for search_string in search_strings[mode]:
                 if mode != 'RSS':
-                    logger.log('Search string: {search}'.format
-                               (search=search_string), logger.DEBUG)
+                    season = episode = None
+                    if isinstance(search_string, tuple):
```

Comment:
I'm talking maintainable accurate code, not theory.  Tuple is all that is used right at this instance.  If that ever changes you have to figure out why it doesn't work (and then would probably do something hacky like force a tuple conversion).  If you do it the way I said it would continue to work correctly with no changes.